### ISSUE TITLE
Blender 2.93.4 and 2.93.3 crash when activating addon

### ISSUE THREAD

**Issue by lucagal** [vicdoval/sverchok-open3d#5]
Hi,
I tried for days to fix the problem, but no way. In Windows 8.1 I installed the weekly build of open3d with python 3.9 and it worked, also if with some interference with other addons, until i made the blender update to 2.93.3 . After that Blender exit always if I try to activate the addon. Previously it strangely made some python script to give errors and then I had to deactivate the addon in Blender  2.93.2.
Any suggestion would be appreciated, thank you

**Comment by vicdoval** [comment]
if you input `import open3d` in the python console does it work?

**Comment by lucagal** [comment]
No, I tried and Blender exit immediately also if i send the command 'import open3d' from it's console. But if I try from the python console ( or terminal) outside Blender the command 'import open3d' works fine.Inviato dal mio Galaxy
-------- Messaggio originale --------Da: Victor Doval ***@***.***> Data: 05/09/21  14:29  (GMT+01:00) A: vicdoval/sverchok-open3d ***@***.***> Cc: lucagal ***@***.***>, Author ***@***.***> Oggetto: Re: [vicdoval/sverchok-open3d] Blender 2.93.4 and 2.93.3 crash when activating addon (#5) 
if you input import open3d in the python console does it work?

**Comment by satabol** [comment]
I have with this situation in Windows:

1. I use Sverchok-Extra addon (with pygalmesh installed) with Sverchok-Open3D (with open3d installed) 
2. Sverchok-open3D activates after Sverchok-Extra addon:

Check addons order activation:
```
for addon in bpy.context.preferences.addons:
	print(addon)
```

![image](https://github.com/vicdoval/sverchok-open3d/assets/14288520/391a5a80-89ce-4c56-90b6-984ced4e92b1)

If "sverchok open3d" is **after** "sverchok-extra" then Blender crash. You need change addons start order. As you can see in this example "sverchok-open3d" addon activates **before** "sverchok extra". To set this order you need to deactivate "sverchok-extra" and activate it again. This will move an "sverchok extra" at the end of list of activation.

If you cannot start Blender then uninstall open3d library manually:

E:\install\Blender\blender-3.6.9-windows-x64\3.6\python\bin\python.exe -m pip uninstall open3d

then try start Blender. Change addons start order and install open3d library again. Then try restart Blender. Let me know is it help?
